bom/openhab-addons/pom.xml

```diff
@@ -338,6 +338,11 @@
       <groupId>org.openhab.addons.bundles</groupId>
       <artifactId>org.openhab.binding.heos</artifactId>
       <version>${project.version}</version>
+    </dependency>
+	<dependency>
```

Comment:
looks like you need to reformat this.

Reply:
Oops.  That's an accidental tab instead of spaces.

I noticed there is also another one before `</dependency>` for org.openhab.binding.bosesoundtouch (line 121 in my branch) that needs to be fixed.